react/src/components/molecules/ArrowNav/index.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { ArrowButton } from '../../../index';
+import './style.css';
+
+const ArrowNav = (props) => {
+  const sectionClasses = classNames({
+    'js-clickable-link': true,
+    'ma__arrow-nav': true,
+    [`ma__arrow-nav--${props.direction}`]: props.direction
+  });
+  const Element = props.href ? 'a' : 'section';
+  const onClickCallback = (e) => {
+    if (typeof props.onClick === 'function') {
+      e.preventDefault();
+      props.onClick(e);
+    }
+  };
+  return(
+    <Element
```

Comment:
now that the whole element is clickable to an href - i think we can get rid of the lower level nested hrefs?

Reply:
Updated and changed the class of the second link so it looked correct.

Reply:
👍